From 67fe9c1b9c6bca92be5e4ebeeea17caa49152030 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:21:54 -0400
Subject: [cpufreq] ondemand: Remove redundant return statement

Message-id: <1409235879-1283-21-git-send-email-prarit@redhat.com>
Patchwork-id: 89360
O-Subject: [RHEL7.1 PATCH BZ 1134639 020/185] cpufreq: ondemand: Remove redundant return statement
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 880eef041655b35f9aa488726ea3c4303a4f2204
Author: Stratos Karafotis <stratosk@semaphore.gr>
Date:   Thu Oct 31 20:27:36 2013 +0200

    cpufreq: ondemand: Remove redundant return statement

    After commit dfa5bb622555 (cpufreq: ondemand: Change the calculation
    of target frequency), this return statement is no longer needed.

    Reported-by: Henrik Nilsson <Karl.Henrik.Nilsson@gmail.com>
    Signed-off-by: Stratos Karafotis <stratosk@semaphore.gr>
    Acked-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq_ondemand.c b/drivers/cpufreq/cpufreq_ondemand.c
index 8f134b3..a24e25a 100644
--- a/drivers/cpufreq/cpufreq_ondemand.c
+++ b/drivers/cpufreq/cpufreq_ondemand.c
@@ -168,7 +168,6 @@ static void od_check_cpu(int cpu, unsigned int load)
    dbs_info->rate_mult =
     od_tuners->sampling_down_factor;
   dbs_freq_increase(policy, policy->max);
-  return;
  } else {
   /* Calculate the next frequency proportional to load */
   unsigned int freq_next;
-- 
1.7.1